3-nodes-mirror-3-dc/inventory/50-inventory.yaml

```diff
@@ -41,3 +47,14 @@ all:
           - static-node-1.ydb-cluster.com
           - static-node-2.ydb-cluster.com
           - static-node-3.ydb-cluster.com
+        # ydbops_local - control cluster from localhost
+        # values: false - use any static host for cluster control
+        #         true  - use localhost for cluster control
+        # Default value: true
+        ydbops_local: true
+        # ydbops_source_bin - binary for ydbops
+        # values: ydbops - Linux x86_64
+        #         ydbops_darwin_arm64 - MacOS Silicon M1/..
+        #         ydbops_darwin_amd64 - MacOS Intel
+        # Default value: ydbops
+        ydbops_source_bin: ydbops
```

Comment:
It's unclear what this setting means: a filepath or some enum?
If it can download it automatically, it'd make more sense to specify a release version or git branch similarly to other executables.

As for the operating system, it would be better to detect it automatically by default.

Reply:
It's impossible to do in case of running ydbops on localhost via 'delagate_to' option.
And we can't run ydbops remotely fully without providing private key on remote host.
Another way to solve it: create a script to detect OS and download appropriate version of ydbops.

Reply:
It looks like Ansible can even detect the OS on its own, see https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_vars_facts.html 

![Screenshot 2024-11-15 at 10 34 35](https://github.com/user-attachments/assets/a392c7c9-677b-48cf-8fd6-8e4f5fb85c70)

Reply:
Yes, it's. But there's an exception - it doesn't work correctly with delegates_to ... But I have one trick in mind .. I'll check it

Reply:
I've found a solution and it's in PR https://github.com/ydb-platform/ydb-ansible/pull/52